runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsPlugin.java

```diff
@@ -213,6 +251,33 @@ public void onMenuEntryAdded(MenuEntryAdded event)
 		}
 	}
 
+	void updateList(String item, boolean hiddenList)
+	{
+		List<String> items = Lists.newArrayList(COMMA_SPLITTER
```

Comment:
Can't you just create copy of hiddenItemList/highlightedItemList here instead of parsing it again? Also, `splitToList`